x/ecocredit/server/query_server.go

```diff
@@ -22,31 +22,22 @@ func (s serverImpl) ClassInfo(ctx context.Context, request *ecocredit.QueryClass
 func (s serverImpl) getClassInfo(ctx sdk.Context, classID string) (*ecocredit.ClassInfo, error) {
 	var classInfo ecocredit.ClassInfo
 	err := s.classInfoTable.GetOne(ctx, orm.RowID(classID), &classInfo)
-	if err != nil {
-		return nil, err
-	}
-
-	return &classInfo, nil
+	return &classInfo, err
```

Comment:
This feels like an unnecessary change. Result should be nil if there's an error

Reply:
Why? If there is an error, then the result by definition in Go SDK is not specified. 
I find it not useful to have the additional 3-4 lines of code just of an err check when the error can be returned directly. I found that google and other Go companies are using this patter across different libraries. Here example in Std lib: https://golang.org/src/strconv/atof.go  line 559.